staff/bernat-casasus/restaurants-app/.versions/restaurants-app-1.0.0/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```

Comment:
el titol!

Reply:
What is "el titol" ? Please If posible use english to send comments!